style/pileup.css

```diff
@@ -119,7 +124,7 @@
 /* pileup track */
 .pileup-root > .pileup {
   flex: 1;  /* stretch to fill remaining space */
-  overflow-y: auto;
+  border-top: 1px solid #666;
```

Comment:
Maybe we should just not have a border? Adding it to all the tracks looks a little excessive (as seen below). Now that the labels stay put as you scroll, you may be able to get away adding some margin between the tracks instead. What do you think?
![border](https://cloud.githubusercontent.com/assets/5685044/8527253/ba81d9b0-23d9-11e5-9518-4b400955ffa8.png)